Gemfile

```diff
@@ -157,7 +157,7 @@ group :test, :development do
   gem "factory_bot_rails", '6.2.0', require: false
   gem 'fuubar', '~> 2.5.1'
   gem 'json_spec', '~> 1.1.4'
-  gem 'knapsack_pro'
+  gem 'knapsack_pro', '7.1.0'
```

Comment:
This will prevent Dependabot from suggesting any more updates. We shouldn't define a version here unless it's necessary.